packages.json

```diff
@@ -0,0 +1,14 @@
+{
+    "packages": {
+        "$c": {
+            "$TRAVIS_BRANCH": {
+                "name": "$c",
```

Comment:
``$c`` is the path to the component source, not the package name of the component